ts/features/wallet/component/__test__/FeaturedCardCarousel.test.tsx

```diff
@@ -350,9 +350,17 @@ describe("FeaturedCardCarousel", () => {
   });
 });
 
-const getComponent = (store: MockStoreEnhanced) =>
-  render(
-    <Provider store={store}>
-      <FeaturedCardCarousel />
-    </Provider>
+// const getComponent = (store: MockStoreEnhanced) =>
+//   render(
+//     <Provider store={store}>
+//       <FeaturedCardCarousel />
+//     </Provider>
+//   );
```

Comment:
```suggestion
```

Can we remove this commented out code block?

Reply:
Thank you! I missed them! Removed in [9131c7a](https://github.com/pagopa/io-app/pull/3958/commits/9131c7a0cf0df4194a286641622cf1410baf049a)